docs/pages/repo/docs/handbook/dev.mdx

```diff
@@ -112,6 +112,36 @@ For example:
 turbo run dev --filter docs
 ```
 
+## Running setup tasks
```

Comment:
This is very focused on `dev` tasks, should we generalize it a bit more?

Reply:
This page is about development tasks. 😄